addon/components/sl-calendar.js

```diff
@@ -113,12 +103,12 @@ export default Ember.Component.extend({
          * Action to trigger component's bound action and pass back content
          * values with dates occurring on the clicked date
          *
-         * @function actions.sendDateContent
-         * @param    {array} dateContent - Collection of content objects with
-         *           date values of the clicked date
-         * @returns  {void}
+         * @function actions:sendDateContent
+         * @param {Array} dateContent - Collection of content objects with
```

Comment:
Do we want a "-" separating the param name and the description? Think this will cause the description to then display as "- ..." in the generated documentation.